lib/src/default_extension_map.dart

```diff
@@ -295,12 +295,13 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'grv': 'application/vnd.groove-injector',
   'grxml': 'application/srgs+xml',
   'gsf': 'application/x-font-ghostscript',
-  'gtar': 'application/x-gtar',
+  'gtar': 'application/gzip',
```

Comment:
Files with extension `gtar` are not standard, but GZip files are.

Reply:
This change is not correct. Not in the scope of the ticket as well
```suggestion
  'gtar': 'application/x-gtar',
```